lua/kubectl/views/init.lua

```diff
@@ -1,10 +1,32 @@
 local ResourceBuilder = require("kubectl.resourcebuilder")
 local actions = require("kubectl.actions.actions")
+local hl = require("kubectl.actions.highlight")
+local tables = require("kubectl.utils.tables")
 
 local M = {}
 
 function M.Hints(hint)
-  actions.floating_buffer(hint, "k8s_hints", { title = "Hints" })
+  local hints = {}
+  local line = hl.symbols.success
```

Comment:
@idelice This was basically what I was rambling about about combining local buffer hints with global ones